src/services/swapi/swapi.js

```diff
@@ -0,0 +1,34 @@
+const swapiEnv = process.env.SW_API_BASE_URL;
+
+export const fetchNameFromAPI = (mode, index) => {
+  const url = `${swapiEnv}/${mode}/${index}`;
+
+  return fetch(url)
```

Comment:
could you please rewrite `fetchNameFromAPI` it into `async/await` with `try/catch` usage
you will learn better (cleaner) approach to write async functions 

https://dmitripavlutin.com/javascript-fetch-async-await/
https://itnext.io/error-handling-with-async-await-in-js-26c3f20bc06a